go/arrow/array/encoded.go

```diff
@@ -68,6 +69,65 @@ func (r *RunEndEncoded) Release() {
 	r.ends.Release()
 }
 
+// LogicalValuesArray returns an array holding the values of each
+// run, only over the range of run values inside the logical offset/length
+// range of the parent array.
```

Comment:
Would be nice to show an example in the docstring, IMHO.

Reply:
added